docs/Plugins/webhook.md

```diff
@@ -12,6 +12,12 @@ When you create an Incoming Webhook within DevLake, DevLake generates a unique U
 
 In v0.14+, users can push "incidents" and "deployments" required by DORA metrics to DevLake via Incoming Webhooks.
 
+Webhooks are meant to be used at the lowest level that you want to relate incidents with deployments. For example, if you want to relate incidents at the individual service level, you will need a webhook per service. If you wish to relate incidents at the product level, you will need a webhook for the product. This is because incidents that are reported to the webhook will be related to the last deployment that was reported to the webhook with a timestamp that is before the incident's timestamp.
```

Comment:
> This is because incidents that are reported to the webhook will be related to the last deployment that was reported to the webhook with a timestamp that is before the incident's timestamp.

The incidents will be related to the `last deployment` no matter the deployment is pushed by webhook or collected by plugins with CI/CD entities. E.g. GitHub, GitLab, Jenkins, Bamboo, etc.

Likewise, the `incidents` used to relate to the deployments are not limited to those from the webhook, incidents collected from Jira, PagerDuty and etc. also count.

So, I guess it's better to paraphrase the wording a little bit.